package/CHANGELOG

```diff
@@ -13,9 +13,9 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-mm/dd/17 richardjgowers, rathann, jbarnoud, orbeckst
+mm/dd/17 richardjgowers, rathann, orbeckst
```

Comment:
I thought me and @utkbansal are the only two people with commits in the release right now